src/api/v1/statuses.ts

```diff
@@ -276,6 +277,26 @@ app.post(
         preferSharedInbox: true,
         excludeBaseUris: [new URL(c.req.url)],
       });
+      if (!owner.account.protected) {
+        const acceptedRelays = await db.query.relays.findMany({
+          where: eq(relays.state, "accepted"),
+          with: {
+            relayServerActor: true,
+          },
+        });
+        await fedCtx.sendActivity(
+          { handle },
+          acceptedRelays.map((relay) => ({
+            id: new URL(relay.relayServerActor.iri),
+            inboxId: new URL(relay.relayServerActor.inboxUrl),
+          })),
```

Comment:
Since it's quite repetitive, how about extracting it out to another function?

```ts
export async function getAcceptedRelays(): Promise<Recipient[]> {
  const acceptedRelays = await db.query.relays.findMany({
    where: eq(relays.state, "accepted"),
    with: {
      relayServerActor: true,
    },
  });
  return acceptedRelays.map((relay) => ({
    id: new URL(relay.relayServerActor.iri),
    inboxId: new URL(relay.relayServerActor.inboxUrl),
  }));
}
```

For your information, the `Recipient` type is exported by Fedify:

```ts
import type { Recipient } from "@fedify/fedify";
```